### ISSUE TITLE
Problem in loading model

### ISSUE THREAD

**Issue by lqr41710085** [YouyiSong/Codes-for-Selective-Learning#2]
hello, i notice that in training.py there exists "NetCon.load_state_dict()" ,but i can't find where or how to get the model "Performance_BTCV_50_50_1.pkl", i wonder if you can detail how to reproducing your experiments in readme.

**Comment by YouyiSong** [comment]
Hi, I have read your email. Thanks for your internesting in this work.

It just uses 50% of the BTCV dataset for training and the remaining for testing.

Due to the size limitation, I have not uploaded it. You can train it by using the standard training with the same epochs and learning rate (Adam).

If there are still any probmes, please inform me. Thanks.

**Comment by lqr41710085** [comment]
thank you for your patient reply! i will have a try.
